### DISCUSSION TITLE
copying a .csv file from Desktop to ssh

### DISCUSSION THREAD

**Discussion by j-balkovec** [community/community#54591] [Programming Help] [answered]
### Body

Hi, I got banned from StackOverflow because my questions were not good enough. I'm not gonna cry about that, but what is stressing me out right now is that my assignment is due 23:59 and it's 16:22 in Seattle right now. I started working on it but I can't copy my file into VSCode. I hope I am posting this question in the right community (I'm sorry if it's the wrong one), but I am desperate for help.

This is my question that got closed on Stack Overflow:
I need to copy a .csv(Comma Separated Values) file from my desktop (running macOS) to an SSH server. I've tried using the scp command, but I keep getting an error saying 

`/Users/jbalkovec/Downloads/WHO-COVID-data.csv: No such file or directory`

 which is strange since all I did was `option clicked` the file and copied the path. 
This is the command I am running in the terminal:

`scp /Users/jbalkovec/Desktop/WHO-COVID-data.csv jbalkovec@cs1.seattleu.edu:/home/st/jbalkovec/2430CPSC/Projects/A4/`

What am I doing wrong? Are there any other methods I could use to transfer the file? 
I tried just dragging it from my desktop to the VS Code editor but unfortunately when I try to open it says `Unauthorized`. When I run the command it asks for my password (since it the University's server) but then it throws an error `No such file or directory` which I am assuming means that bash couldn't find the file right?
I don't really know what tags to put under here, so please don't get mad if some of them are wrong.
Thank you.

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by GeoffreyFClark** [comment]
Double-check the filepath. Is it on your Desktop or in your Downloads? What happens when you run:
`scp /Users/jbalkovec/Downloads/WHO-COVID-data.csv jbalkovec@cs1.seattleu.edu:/home/st/jbalkovec/2430CPSC/Projects/A4/`

**Reply by airtower-luna** [reply]
That, and also check if the target directory exists.

**Reply by j-balkovec** [reply]
I tried running rsync and scp (I've done this in the past) and I am still having trouble. When I run the scp command I get this /Users/jbalkovec/Downloads/WHO-COVID-data.csv: No such file or directory.

I attached a SC of my desktop and my terminal if that helps in any way. Is my spacing wrong perhaps?

<img width="1440" alt="Screenshot 2023-05-05 at 16 45 47" src="https://user-images.githubusercontent.com/116223977/236585763-73ec9efe-cd6b-48ba-8015-ada7a6d0492e.png">

**Reply by j-balkovec** [reply]
The screenshot from my monitor is not working for some reason.
![Screenshot 2023-05-05 at 16 50 13 (2)](https://user-images.githubusercontent.com/116223977/236586000-30efd2bb-038a-40df-82b3-460bc0087415.png)

**Reply by airtower-luna** [reply, marked as answer]
The terminal in that screenshot looks like you're actually logged in on the `cs1.seattleu.edu` server over SSH. So you're running the `scp` command on that host (where the CSV isn't yet) instead of on your local machine. Run it locally and it should work, assuming the paths are correct and you can log in.

**Reply by j-balkovec** [reply]
Thank you, that worked. I got one more question. I opened the file and there is supposed to be a gazillion lines of data, but in my case there is only one line that says "Unauthorized". Is that a thing with .csv files or is that something my professor should fix?

**Reply by airtower-luna** [reply]
If that's all that's in the file something is definitely wrong. I can't say who needs to fix it (I don't know where the file is from, how you got it, etc.), but you should definitely look into getting a correct one, from your professor or otherwise.

**Reply by j-balkovec** [reply]
Yeah, I looked it up online and it seems to be a Safari thing. I tried using Tor and it worked. Thank you for all the help
